lobe_android/app/src/main/java/com/example/test/tflite/Classifier.java

```diff
@@ -122,14 +122,8 @@
    */
   public static Classifier create(Activity activity, Model model, Device device, int numThreads)
       throws IOException {
-    if (model == Model.QUANTIZED_MOBILENET) {
-      return new org.tensorflow.lite.examples.classification.tflite.ClassifierQuantizedMobileNet(activity, device, numThreads);
-    } else if (model == Model.FLOAT_MOBILENET) {
-      return new ClassifierFloatMobileNet(activity, device, numThreads);
-    } else if (model == Model.FLOAT_EFFICIENTNET) {
-      return new ClassifierFloatEfficientNet(activity, device, numThreads);
-    } else if (model == Model.QUANTIZED_EFFICIENTNET) {
-      return new ClassifierQuantizedEfficientNet(activity, device, numThreads);
+        if (model == Model.FLOAT_MOBILENET) {
```

Comment:
do we need to have this if statement?

Reply:
Probably not right now -- I left it as-is for potential OD support.

Reply:
I'd rather have the property be Model.IMAGE_CLASSIFICATION then - FLOAT_MOBILENET seems confusing unless you know what that means

Reply:
Ah -- yes -- absolutely can and should change that. Will do!

Reply:
Done!